packages/wdio-utils/src/shim.js

```diff
@@ -53,24 +54,36 @@ let runFnInFiberContext = function (fn) {
     }
 }
 
-let wrapCommand = async function (commandName, fn, ...args) {
-    await executeHooksWithArgs.call(this, this.options.beforeCommand, [commandName, args])
+let wrapCommand = function wrapCommand (commandName, fn) {
+    return async function wrapCommandFn (...args) {
+        const beforeHookArgs = [commandName, args]
+        if (!inCommandHook) {
+            inCommandHook = true
+            await executeHooksWithArgs.call(this, this.options.beforeCommand, beforeHookArgs).catch((err) => err)
```

Comment:
Are you sure we need to catch error? Shouldn't  executeHooksWithArgs catch all errors for us?